server/repository/src/migrations/v2_00_00/assets/asset_log.rs

```diff
@@ -2,6 +2,30 @@ use crate::migrations::DATETIME;
 use crate::{migrations::sql, StorageConnection};
 
 pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    #[cfg(feature = "postgres")]
+    sql!(
+        connection,
+        r#"
+          CREATE TYPE asset_log_status AS ENUM (
+            'NOT_IN_USE',
+            'FUNCTIONING',
+            'FUNCTIONING_BUT_NEEDS_ATTENTION',
+            'NOT_FUNCTIONING',
+            'DECOMMISSIONED'
+          );
+          CREATE TABLE asset_log (
+            id TEXT NOT NULL PRIMARY KEY,
+            asset_id TEXT NOT NULL REFERENCES asset(id),
+            user_id TEXT NOT NULL REFERENCES user_account(id),
+            status asset_log_status,
+            reason_id TEXT REFERENCES asset_log_reason(id),
+            comment TEXT,
+            type TEXT,
+            log_datetime {DATETIME} NOT NULL
+          );
+        "#
+    )?;
```

Comment:
I think we probably want to do all these migration in the  `pg_enums::migrate` function and put it in to the 2.1 migration? We don't want to assume this will be in for 2.0 installs right?

Reply:
Yes I just realized it on the weekend as well that I was too optimistic to get it into 2.0. Will move everything over.